dcor/tests/test_homogeneity.py

```diff
@@ -119,3 +119,30 @@ def test_different_distributions(self):
             a, b, num_resamples=num_resamples, random_state=random_state)
 
         self.assertLess(result.p_value, significance)
+
+    def test_different_means_median(self):
+        """
+        Test that the test works on different distributions, using the median average.
+
+        As the distributions are not the same, the test should reject
+        the null hypothesis.
+
+        """
+        num_samples = 100
+
+        random_state = np.random.RandomState(0)
+
+        a = random_state.normal(loc=0, size=(num_samples, 1))
+        b = random_state.normal(loc=1, size=(num_samples, 1))
+
+        significance = 0.01
+        num_resamples = int(3 / significance + 1)
+
+        median_result = dcor.homogeneity.energy_test(a, b, num_resamples=num_resamples, random_state=random_state, average=np.median)
```

Comment:
Please, lines should not be larger than 79 characters, according to PEP8.

Reply:
Will do. Would you consider using Black to auto format?

Reply:
If you mean autoformat as a commit action then no, I do not think that is a good idea to change code behind developer's back. I am open to use a Github action that flags format issues, such as [wemake-python-styleguide](https://github.com/wemake-services/wemake-python-styleguide). I have used it in other project and it works well after you configure the rules to ignore properly. And, of course, if you want to use autoformatters in your local machine you can do it, but review the code before commiting it, and try not to format parts of the code unrelated to the changes in order to prevent conflicts and make the changes easier to spot.

Reply:
Fair enough. I will manually format the code.